lib/constants.js

```diff
@@ -94,6 +94,31 @@ module.exports = {
         "claim": 600
     },
 
+    RANGE_ATTACK: 1,
+    RANGE_ATTACK_CONTROLLER: 1,
+    RANGE_BOOST_CREEP: 1,
+    RANGE_BUILD: 3,
+    RANGE_CLAIM_CONTROLLER: 1,
+    RANGE_DISMANTLE: 1,
+    RANGE_DROP: 1,
```

Comment:
I agree, RANGE_DROP seems weird. You can not give a position to where you'd want to drop resources.